src/examples/cmake/template/toolchain.cmake.in

```diff
@@ -91,9 +94,35 @@ include("${_INTERNAL_ANDROID_STUDIO_ORIGINAL_CMAKE_TOOLCHAIN_FILE}")
 
 # STEP 3: Tweak CMake Android variables
 
-set(CMAKE_SYSTEM_VERSION "${ANDROID_NATIVE_API_LEVEL}")
```

Comment:
CMake internal Android related code will not be loaded if you will remove this line. E.g. variable [CMAKE_ANDROID_ARCH](https://cmake.org/cmake/help/latest/variable/CMAKE_ANDROID_ARCH.html) will not be set, you can't use `CMAKE_SYSTEM_VERSION` to analyze Android API (it will be set to `1` always). Effectively it removing [Tweak variables to match CMake 3.7.1+ format](https://github.com/elucideye/drishti/blob/7001ac0f6e8e5f9a04a8eae70274a613a13ce96b/src/examples/cmake/template/toolchain.cmake.in#L33-L37) part of this toolchain.

These variables are important because they can be used by third parties like OpenCV:
- https://github.com/hunter-packages/opencv/commit/6b1ca6faa66ea0c2a64c2c51781a6873de90e774

Reply:
Thanks for the review.  From the link you sent, it seems any successful build with these workarounds will be lacking NEON acceleration in the OpenCV components (at least).